Deployment/README.md

```diff
@@ -47,6 +47,14 @@ The following are all valid invocations:
 You can find the output rap in the Deployment/Packages folder.
 *Note:* the output rap will override any other raps with the same version.
 
+## Creating a Nuget Package
+
+The `Milyli.ScriptRunner.Services.Interfaces` and `Milyli.ScriptRunner.Services.Interfaces.Contracts` projects comprise a consumable package of [Kepler services](../Milyli.ScriptRunner.Services.Interfaces/) that allow other applications to use ScriptRunner.  These can be packed into a .nupkg file using the `CreateNugetPackage` option as a -TaskList parameter, or alternatively the `CreateRapAndNuget` option to create both a Rap and Nupkg.
```

Comment:
`CreateRapAndNuget` should probably be it's own section. 

  - [ ] Create a section per "external" psake task